File: Client/src/app/models/import-file.ts
export class ImportFile {
    createdOn: Date;
    fileExtension: string;
    fileName: string;
    importFileId: number;
    fileSize: string;
    fileType: string;
    processed: any;
    fileDate: Date | string | null;
    constructor(createdOn: Date, fileExtension: string, importFileId: number,
                fileName: string, fileSize: string, fileType: string, processed: any, fileDate: Date | string | null) {
      this.createdOn = createdOn;
      this.fileExtension = fileExtension;
      this.importFileId = importFileId;
      this.fileName = fileName;
      this.fileSize = fileSize;
      this.fileType = fileType;
      this.processed = processed;
      this.fileDate = fileDate;
    }
}

File: Server/DAL/BridgeportClaims.Entities/DomainModels/ImportFile.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace BridgeportClaims.Entities.DomainModels
{
    public class ImportFile
    {
        [Required]
        public virtual int ImportFileId { get; set; }
        [Required]
        public virtual ImportFileType ImportFileType { get; set; }
        [Required]
        public virtual byte[] FileBytes { get; set; }
        [Required]
        [StringLength(255)]
        public virtual string FileName { get; set; }
        [StringLength(30)]
        public virtual string FileExtension { get; set; }
        [Required]
        [StringLength(100)]
        public virtual string FileSize { get; set; }
        [Required]
        public virtual bool Processed { get; set; }
        [Required]
        public virtual DateTime CreatedOnUtc { get; set; }
        [Required]
        public virtual DateTime UpdatedOnUtc { get; set; }
    }
}
